sdk/core/core-http/README.md

```diff
@@ -23,15 +27,51 @@ Examples can be found in the `samples` folder.
 
 ## Next steps
 
+First obtain an access token to the management service used in the sample code
+
+One easy way to get an access token is using [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest)
+
+1. Sign in
+```shell
+az login
+```
+2. Select the subscription to use
+```shell
+az account set -s <subscription id>
+```
+3. Obtain an access token
+```shell
+az account get-access-token --resource=https://management.azure.com
+```
+
 ### Node.js
 
-- Set the subscriptionId and token
-- Run `node samples/node-sample.js`
+- Set environment variables used in the sample code
+
+on \*Nix/MacOS
+```shell
+export AZURE_SUBSCRIPTION_ID=<subscription id>
+export ACCESS_TOKEN=<access token>
+```
+or on Windows,
+
+```shell
+set AZURE_SUBSCRIPTION_ID=<subscription id>
+set ACCESS_TOKEN=<access token>
+```
+
+- Change directory to samples folder, compile the TypeScript code, then run the sample
+
+```
+cd samples
+tsc node-sample.ts
+node node-sample.js
+```
 
-### In the browser
+### Browser
 
-- Set the subscriptionId and token and then run
-- Open index.html file in the browser. It should show the response from GET request on the storage account. From Chrome type Ctrl + Shift + I and you can see the logs in console.
+- Set values of `subscriptionId` and `token` variable in `samples/index.js`
```

Comment:
In case of the browser sample, we are asking them to update the subscription id and token directly in the sample file, but in case of node, we are asking them to set as env variables. Why the difference?

Reply:
It was like this in the old code. I can make node case the same as browser.

Reply:
Let's do that. The two need to have similar instructions